docs/cedarling/reference/benchmarks.md

```
@@ -2,9 +2,98 @@
      (cross_binding_bench_summary job). Do not edit by hand;
-     changes are overwritten on the next run on main. -->
+     changes are overwritten on the next publish run. -->
 
 Cross-platform authorization benchmarks for the Cedarling bindings,
-regenerated on pushes to `main` that change `jans-cedarling/**`.
+regenerated on pushes to `main` that change `jans-cedarling/**`,
+or on a manual `workflow_dispatch` run against `main`.
+
+_Last updated: 2026-08-19 12:56 UTC (commit f7a237e)._
+
+### Cross-Platform Binding Benchmarks
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)